aws/vpc-public-only/outputs.tf

```diff
@@ -1,23 +1,24 @@
 output "id" {
-  value = "${aws_vpc.vpc.id}"
+  value = aws_vpc.vpc.id
 }
 
 output "vpc_default_sg_id" {
-  value = "${data.aws_security_group.vpc_default_sg.id}"
+  value = data.aws_security_group.vpc_default_sg.id
 }
 
 output "public_subnet_ids" {
-  value = ["${aws_subnet.public_subnet.*.id}"]
+  value = aws_subnet.public_subnet.*.id
 }
 
 output "public_subnet_cidr_blocks" {
-  value = ["${aws_subnet.public_subnet.*.cidr_block}"]
+  value = aws_subnet.public_subnet.*.cidr_block
 }
 
 output "db_subnet_group_name" {
-  value = "${aws_db_subnet_group.db_subnet_group.name}"
+  value = aws_db_subnet_group.db_subnet_group.name
 }
 
 output "aws_zones" {
-  value = ["${var.aws_zones}"]
+  value = [var.aws_zones]
```

Comment:
Similar to earlier question: If `var.aws_zones` is already a list, does it still need the square brackets?

Reply:
Good catch.  No, the square brackets are not needed.  Also true for aws/vpc/outputs.tf.  Although I use vpc, I don't use the output so the error didn't show up.